common/dm.c

```diff
@@ -77,6 +78,27 @@ dm_get_by_name(const char *name)
 	panic("dm: Device %s not found", name);
 }
 
+int
+dm_set_pins(struct device *dev, uint8_t num_pins)
```

Comment:
Please make this `dm_setup_pins` (so it can match a future `dm_setup_clocks`, etc.)

Also, if you don't mind, please put this below `dm_init`. Beyond being alphabetical, it allows `dm_init` to be closer to its helper function.

Reply:
Done.